src/usr/local/buildpack/tools/v2/node.sh

```diff
@@ -40,8 +40,17 @@ function install_tool () {
   local versioned_tool_path
   local npm # temp npm executable
 
+  checksums=$(get_from_url "https://nodejs.org/dist/v${TOOL_VERSION}/SHASUMS256.txt")
+
+  # get checksum from file
+  original_checksum=$(grep "node-v${TOOL_VERSION}-linux-x64.tar.xz" "${checksums}" | cut -d' ' -f1)
```

Comment:
please move the `linux-x64` to a local variable, will need that for arm64 support. checkout the open draft PR.